twilight-mention/src/fmt.rs

```diff
@@ -189,13 +282,45 @@ impl Mention<Id<UserMarker>> for User {
     }
 }
 
+/// Components to construct a slash command mention.
+///
+/// Format slash commands, subcommands and subcommand groups.
+/// See [Discord Docs/Message Formatting].
+/// See [Discord Docs Changelog/Slash Command Mentions].
+///
+/// [Discord Docs/Message Formatting]: https://discord.com/developers/docs/reference#message-formatting
+/// [Discord Docs Changelog/Slash Command Mentions]: https://discord.com/developers/docs/change-log#slash-command-mentions
+#[allow(missing_docs)]
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub enum CommandMention {
+    // Field order is the same as the mention format.
```

Comment:
i think we should still order fields alphabetically like we do (almost) everywhere

Reply:
I ordered them now.